Can't create new profile

At HEAD of master:
- Start PanelPro, having it set to allow profile selection
- Click "New" and create a new one
- Continue to start

You get the JMRI main screen, Help opens to the preferences help page, then instead of preferences opening you get a crash window:
<img width="1283" alt="banners_and_alerts_and_unrecoverable_error_encountered" src="https://user-images.githubusercontent.com/2774117/28549424-c67058d8-708e-11e7-947c-3775dcfffe78.png">

Clicking OK closes the program.  There's no way to use that profile.  I've attached a .zip version of it.

[MERG_CBus_Simulator.zip](https://github.com/JMRI/JMRI/files/1171517/MERG_CBus_Simulator.zip)

Console traceback for the above:
```
     [java] 2017-07-24 16:37:36,837 hrottle.AbstractWiThrottlePreferences INFO  - Could not find WiThrottle preferences file (/Users/jake/Library/Preferences/JMRI/MERG_CBus_Simulator/throttle/WiThrottlePreferences.xml).  Normal if preferences have not been saved before. [init prefs]
     [java] 2017-07-24 16:37:37,116 ptionhandler.UncaughtExceptionHandler ERROR - Uncaught Exception caught by jmri.util.exceptionhandler.UncaughtExceptionHandler [init prefs]
     [java] java.util.ServiceConfigurationError: jmri.swing.PreferencesPanel: Provider jmri.web.server.FrameServletPreferencesPanel could not be instantiated
     [java] 	at java.util.ServiceLoader.fail(ServiceLoader.java:232)
     [java] 	at java.util.ServiceLoader.access$100(ServiceLoader.java:185)
     [java] 	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:384)
     [java] 	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
     [java] 	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
     [java] 	at apps.gui3.TabbedPreferences.init(TabbedPreferences.java:178)
     [java] 	at apps.Apps$3.lambda$run$4(Apps.java:377)
     [java] 	at apps.Apps$3$$Lambda$58/1823985821.accept(Unknown Source)
     [java] 	at java.util.Optional.ifPresent(Optional.java:159)
     [java] 	at apps.Apps$3.run(Apps.java:376)
     [java] 	at java.lang.Thread.run(Thread.java:745)
     [java] Caused by: java.lang.NullPointerException: Required nonnull default for jmri.web.server.WebServerPreferences does not exist.
     [java] 	at java.util.Objects.requireNonNull(Objects.java:228)
     [java] 	at jmri.InstanceManager.getDefault(InstanceManager.java:184)
     [java] 	at jmri.web.server.FrameServletPreferencesPanel.initComponents(FrameServletPreferencesPanel.java:71)
     [java] 	at jmri.web.server.FrameServletPreferencesPanel.<init>(FrameServletPreferencesPanel.java:57)
     [java] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
     [java] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
     [java] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
     [java] 	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
     [java] 	at java.lang.Class.newInstance(Class.java:438)
     [java] 	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:380)
     [java] 	... 8 more
```

--- a/java/src/jmri/web/server/WebServerPreferences.java
+++ b/java/src/jmri/web/server/WebServerPreferences.java
@@ -8,6 +8,7 @@ import java.util.Arrays;
 import java.util.prefs.BackingStoreException;
 import java.util.prefs.Preferences;
 import jmri.InstanceManager;
+import jmri.InstanceManagerAutoDefault;
 import jmri.beans.PreferencesBean;
 import jmri.jmrit.XmlFile;
 import jmri.profile.ProfileManager;
@@ -23,7 +24,7 @@ import org.slf4j.LoggerFactory;
 /**
  * @author Randall Wood Copyright (C) 2012
  */
-public class WebServerPreferences extends PreferencesBean {
+public class WebServerPreferences extends PreferencesBean implements InstanceManagerAutoDefault {
 
     // preferences elements
     public static final String DISALLOWED_FRAMES = "disallowedFrames"; // NOI18N
@@ -150,10 +151,15 @@ public class WebServerPreferences extends PreferencesBean {
         this.readPreferences(sharedPreferences);
     }
 
+    /**
+     * Return the current default WebServerPreferences object.
+     * 
+     * @deprecated 4.9.2 in favor of use of InstanceManager, which will 
+     *              automatically instantiate one of these if needed (see {@link InstanceManagerAutoDefault})
+     */
+    @Deprecated
     public static WebServerPreferences getDefault() {
-        return InstanceManager.getOptionalDefault(WebServerPreferences.class).orElseGet(() -> {
-            return InstanceManager.setDefault(WebServerPreferences.class, new WebServerPreferences());
-        });
+        return InstanceManager.getDefault(WebServerPreferences.class);
     }
 
     private void readPreferences(Preferences sharedPreferences) {

